extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -84,22 +86,42 @@ require([
 				};
 
 			if (ooyalaAdSetProvider.canShowAds()) {
-				options.adSet = ooyalaAdSetProvider.get(1, correlator, {
-					contentSourceId: videoData.dfpContentSourceId,
-					videoId: videoId
-				});
 				options.replayAds = ooyalaAdSetProvider.adsCanBePlayedOnNextVideoViews();
+
+				if (a9) {
+					a9.waitForResponse()
+						.then(function () { return a9.getSlotParams('FEATURED'); })
```

Comment:
Are you sure those slot params are enough for A9 video targeting? I see documentation is advising us to use `qsParams`.

Reply:
There is no `qsParams` parameter :disappointed: 
![image](https://user-images.githubusercontent.com/2962313/29813668-0e7c8e0c-8cab-11e7-8693-c5bc018c5c50.png)

Reply:
We talked offline - there is `qsParams` attribute ;)
Code is fixed.
For now, let's use `getSlotParams()` and see how it works while QA.